### ISSUE TITLE
How to train a model that starts from a previously trained checkpoint?

### ISSUE THREAD

**Issue by sstamatis01** [nameless-Chatoyant/PWC-Net_pytorch#5]
Hello again,

I have finished training my model from Flying chairs for 400k steps with learning rate 1e-4. Now I would like to train the same dataset for more steps from that checkpoint but change the learning rate. How can I initialize my model to get the weights from that checkpoint?

Thanks in advance.

**Comment by nameless-Chatoyant** [comment]
Hi @sstamatis01 

Try something like
```
python3 main.py trian --load <model file> --lr <new lr>
```

Note that only weights not full networks are saved, which means you'd better keep other arguments still. For example, `--residual`, `--batch_norm`, `--input_norm`, etc.

**Comment by sstamatis01** [comment]
Yes this is working, Thanks for quick reply!

On a side note is it possible to reproduce the data augmentation scheme of the original PWC-Net paper? I believe it it the same as Flownet2
